apps/astarte_realm_management/test/support/database_test_helper.exs

```diff
@@ -305,4 +397,288 @@ defmodule Astarte.RealmManagement.DatabaseTestHelper do
   def jwt_public_key_pem_fixture do
     @jwt_public_key_pem
   end
+
+  def seed_individual_datastream_test_data!(
+        realm_name,
+        device_id,
+        interface_name,
+        major,
+        endpoint,
+        path
+      ) do
+    interface_id = CQLUtils.interface_id(interface_name, major)
+    endpoint_id = CQLUtils.endpoint_id(interface_name, major, endpoint)
+
+    Xandra.Cluster.run(:xandra, fn conn ->
+      statement = """
+      INSERT INTO #{realm_name}.individual_datastreams
+        (device_id, interface_id, endpoint_id, path, value_timestamp, reception_timestamp, reception_timestamp_submillis, integer_value)
+      VALUES (:device_id, :interface_id, :endpoint_id, :path, '2017-09-28 04:06+0000', '2017-09-28 05:06+0000', 0, 42);
```

Comment:
Pass also the values you insert here as a parameter instead of hardcoding them, otherwise the known value magically appears in assertions in the test case and it's not clear where it is coming from.